OrbitGl/TimeGraph.cpp

```diff
@@ -318,7 +312,12 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const FunctionInfo* fu
       ++thread_count_map_[timer_info.thread_id()];
     } else {
       scheduler_track_->OnTimer(timer_info);
-      cores_seen_.insert(timer_info.processor());
+      if (!cores_seen_.count(timer_info.processor())) {
```

Comment:
I feel that this cores_seen_ is a bit unintuitive. Couldn't we just have a `num_cores_` variable, and here check `if(num_cores_ < timer_info.processor()) num_cores_ = timer_info.processor(); ....`? 
But anyways. Other than that, it looks good. Also this is just a nit. Feel free to ignore.

Reply:
Oh, I didn't suppose that the processors went from 0 to N-1. Is it a guarantee?

Reply:
Yes, it is.